src/java.base/share/classes/java/lang/invoke/InnerClassLambdaMetafactory.java

```diff
@@ -89,17 +85,19 @@
     private static final AtomicInteger counter = new AtomicInteger();
```

Comment:
This counter could be removed now, it looks like.

Reply:
thanks for catching it. will fix.